### ISSUE TITLE
Update 'the latest stable JAR' in README

### ISSUE THREAD

**Issue by chenzimin** [SpoonLabs/gumtree-spoon-ast-diff#57]
Using the JAR from 'the latest stable JAR' in REAMDE cause the following exception:

```
Exception in thread "main" java.lang.NoClassDefFoundError: com/github/gumtreediff/tree/TreeContext
	at gumtree.spoon.builder.SpoonGumTreeBuilder.<init>(SpoonGumTreeBuilder.java:34)
	at gumtree.spoon.AstComparator.compare(AstComparator.java:91)
	at gumtree.spoon.AstComparator.compare(AstComparator.java:77)
	at se.kth.statementDiff.App.main(App.java:32)
Caused by: java.lang.ClassNotFoundException: com.github.gumtreediff.tree.TreeContext
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:335)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 4 more
```

However, building and using JAR (gumtree-spoon-ast-diff-1.6-SNAPSHOT-jar-with-dependencies.jar) from source is fine.

**Comment by martinezmatias** [comment]
Hi @chenzimin 
Thanks for reporting the problem. I will take a lot to that problem.
Regards
Matias

**Comment by monperrus** [comment]
fixed in 054809beee80e5180aa197aff38a64a1691465d9
